sw/device/tests/penetrationtests/firmware/fi/ibex_fi.c

```diff
@@ -151,6 +151,7 @@ OT_SECTION(".data")
 static volatile uint32_t sram_main_buffer[256];
 
 // Make sure that this function does not get optimized by the compiler.
+OT_USED
 void increment_counter(void) __attribute__((optnone)) {
   asm volatile("addi x5, x5, 1");
 }
```

Comment:
With the `OT_USED` I think you can now drop the `__attribute__((optnone))`, as I guess the attribute was just trying to prevent this function from being optimized away, since it wasn't referenced outside of the assembly code string.